Resources/Prototypes/Objectives/traitor.yml

```
@@ -333,3 +333,25 @@
     stealGroup: SupermatterSliver
     objectiveNoOwnerText: objective-condition-steal-smsliver-title
     descriptionText: objective-condition-steal-smsliver-description
+
+## ntrep
+
+- type: entity
+  abstract: true
+  parent: BaseTraitorStealObjective
+  id: BaseNTRepObjective
+  components:
+  - type: Objective
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)